topiary-cli/src/error.rs

```diff
@@ -167,3 +167,13 @@ impl From<tokio::task::JoinError> for TopiaryError {
         )
     }
 }
+
+// Tells whether an error should raise a message on stderr,
+// or if it's an "expected" error.
+#[allow(clippy::match_like_matches_macro)]
+pub fn benign(err: &TopiaryError) -> bool {
+    match err {
+        TopiaryError::Lib(FormatterError::PatternDoesNotMatch) => true,
+        _ => false,
+    }
+}
```

Comment:
This looks good, but I wonder if it would be more "Rusty" (FWIW) to put this in an `impl TopiaryError`.